scripts/demangle.py

```diff
@@ -59,14 +77,64 @@ def demangle(func: str):
     #     # Demangling failed
     #     print("Demangling failed with status", status)
 
-for json_file_name in file_names:
+def get_source_code(file_path, file_name):
+    # get source code from docker
+    # # Create a container from the image
+    client = docker.from_env()
+    image = client.images.get('gcr.io/oss-fuzz/'+proj_name+':latest')
+    container=client.containers.run(image,detach=True)
+    # The docker does not support regular expression, brute force try
+
+    #check file_path, it may not the absolut path
+    if not '/' in file_path:
+        file_path='/src/'+proj_name+'/'+file_path
+    try:
+        code_content = ''
+        for chunk in container.get_archive(file_path)[0]:
+            code_content+=chunk.decode('utf-8')
+    except:
+        code_content='ERROR'
+        print('ERROR: File '+file_path+" NOT found")
+        print()
```

Comment:
Add a `\n` at the end of your string or use `end='\n\n'` instead of `print` twice.